lecli/api_key/api.py

```diff
@@ -0,0 +1,126 @@
+import json
+import sys
+
+import requests
+
+from lecli import api_utils
+from lecli import response_utils
+
+
+def _url(api_key_id=None):
+    """
+    Get rest query "path" and "url" respectively
+    """
+    nodes = ['management', 'accounts', api_utils.get_account_resource_id(), 'apikeys']
+    if api_key_id:
+        nodes.append(api_key_id)
+
+    return api_utils.build_url(nodes)
+
+
+def handle_api_key_response(response):
+    """
+    Handle get api key response
+    """
+    if response_utils.response_error(response):
+        sys.exit(1)
+    elif response.status_code in [200, 201]:
+        api_utils.pretty_print_string_as_json(response.text)
+
+
+def delete(api_key_id):
+    """
+    Delete an api key with the provided ID
+    """
+    action, url = _url(api_key_id)
+    headers = api_utils.generate_headers('owner', method='DELETE', body='', action=action)
+
+    try:
+        response = requests.delete(url, headers=headers)
+        if response_utils.response_error(response):
+            sys.stderr.write('Deleting api key failed, status code: %d' % response.status_code)
+            sys.exit(1)
+        elif response.status_code == 204:
+            sys.stdout.write('Deleted api key with id: %s \n' % api_key_id)
```

Comment:
What happens when this branch is false, and it's not a 204? If there's no possibility for another status code we can just have an else here and maybe a comment to say the status is 204